core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -1,137 +1,853 @@
 <div class="teach-page">
-  <div class="oppia-static-header">
-    <h1 [innerHTML]="'I18N_TEACH_PAGE_HEADING' | translate"></h1>
+  <div class="oppia-teach-section-one" dir="auto">
+
+    <div class="oppia-teach-section-one-left"  *ngIf="!isWindowNarrow">
+      <h1 class="oppia-teach-h1" [innerHTML]="'I18N_TEACH_PAGE_HEADING' | translate"></h1>
+      <div class="oppia-teach-h2-container">
+        <h2 *ngIf="!isWindowNarrow" class="oppia-teach-h2 oppia-teach-h2-text" [innerHTML]="'I18N_TEACH_PAGE_CONTENT' | translate"></h2>
+      </div>
+      <div class="oppia-teach-button-container">
+        <form action="classroomUrl">
+          <button type="submit" (click)="onClickStartLearningButton()"
+                  class="btn oppia-teach-button-one oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_TEACH_PAGE_ACTION_START_LEARNING' | translate">
+          </button>
+        </form>
+      </div>
+    </div>
+
+    <div class="oppia-teach-section-one-left" *ngIf="isWindowNarrow">
+      <h1 class="oppia-teach-h1 mobile-heading" [innerHTML]="'I18N_TEACH_PAGE_HEADING' | translate"></h1>
+      <h2  class="oppia-teach-h2 oppia-teach-h2-text mobile-h2" [innerHTML]="'I18N_TEACH_PAGE_CONTENT' | translate"></h2>
+      <div class="oppia-teach-button-container mobile-button">
+        <form action="classroomUrl">
+          <button type="submit" (click)="onClickStartLearningButton()"
+                  class="btn oppia-teach-button-one oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_TEACH_PAGE_ACTION_START_LEARNING' | translate">
+          </button>
+        </form>
+      </div>
+    </div>
+
+    <div *ngIf="!isWindowNarrow" class="oppia-teach-section-one-right">
+      <picture class="oppia-teach-main-image image-desktop">
+        <source type="image/webp" [src]="getStaticImageUrl('/teach/student.webp')">
+        <source type="image/png" [src]="getStaticImageUrl('/teach/student.png')">
```

Comment:
```suggestion
        <source type="image/webp" [srcset]="getStaticImageUrl('/teach/student.webp')">
        <source type="image/png" [srcset]="getStaticImageUrl('/teach/student.png')">
```

ditto in other places

Reply:
Done, It causes images to disappear in my system for section five(testinomials part). Elsewhere it works fine. Can you please have a look?